api/src/services/application-csv-export.service.ts

```diff
@@ -727,6 +589,18 @@ export class ApplicationCsvExporterService
       );
     }
 
+    // if its for the lottery insert the lottery position
+    if (forLottery) {
+      headers.unshift({
+        path: 'applicationLotteryPositions',
+        label: 'Lottery Position',
```

Comment:
The sample file calls this column "Raw Lottery Rank". Talking to Sarah it sounds like we want to have it be that instead of "Lottery Position"